sdk/storage/Azure.Storage.Files.DataLake/src/ErrorExtensions.cs

```diff
@@ -27,12 +28,11 @@ internal static Exception CreateException(this string jsonMessage, ClientDiagnos
             }
             else
             {
-                Dictionary<string, Dictionary<string, string>> errorDictionary
-                    = JsonSerializer.Deserialize<Dictionary<string, Dictionary<string, string>>>(jsonMessage);
+                DataLakeErrorWrapper error = JsonSerializer.Deserialize<DataLakeErrorWrapper>(jsonMessage);
```

Comment:
We should be using `JsonDocument.Parse` instead of `JsonSerializer`.

Reply:
Per https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-json-use-jsondocument

Reply:
fixed.

Reply:
Awesome - that `property.Value.ToString()` is a really nice touch in case the service ever starts sending back more complicated values.